internal/driver/glfw/menu_bar.go

```diff
@@ -16,11 +16,41 @@ var _ fyne.Widget = (*MenuBar)(nil)
 // MenuBar is a widget for displaying a fyne.MainMenu in a bar.
 type MenuBar struct {
 	widget.Base
-	Items []fyne.CanvasObject
-
+	Items      []fyne.CanvasObject
+	index      int
 	active     bool
-	activeItem *menuBarItem
 	canvas     fyne.Canvas
+	activeItem *menuBarItem
+}
+
+// HandleKey will process keys pressed when menu is open
+func (b *MenuBar) HandleKey(key fyne.KeyName) {
```

Comment:
This should not be an exported API

Reply:
Ok

Reply:
By way of explanation here (sorry it was lacking before) we don't add a new exported API unless there is a solid use-case for developers.